### ISSUE TITLE
Web Adaptor fails to install with exit code 1642

### ISSUE THREAD

**Issue by BrendanEsri** [Esri/arcgis-powershell-dsc#523]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Module Version

* 4.2.0

### Affected Resource(s)

* ArcGIS Web Adaptor

### Configuration Files

* N/A

### Expected Behavior

* Installs Web Adaptor

### Actual Behavior

* Web Adaptor fails to install with exit code 1642

### Steps to Reproduce

* **Issue:** When spinning a new box up sometimes it comes with older versions of .NET, if it does then run the Invoke command and Web Adaptor will fail to install as it is unable to install the .NET 6.0.9 (or whatever version you specify in the json) but the logs in the DSC output do not specify this.

* **Workaround:** Uninstall all the mentions of the old version of .NET (in our case it was 6.0.25) and run PowerShell DSC Invoke again and it will install the 6.0.9 (or whatever version you specify in the json successfully then successfully install Web Adaptor

### Important Factoids

* N/A

### References

* N/A

**Comment by BrendanEsri** [comment]
Just opened this for reference if others run into the same thing. Potentially put some messaging in PowerShell DSC ArcGIS Module logs to tell us to check if other versions of .NET are installed. Issue is resolved though through the provided workaround

**Comment by lachlanwainwright** [comment]
Hi @BrendanEsri 

I encountered this exact same issue, thanks for the workaround.

I grabbed the .net bundle installer from https://dotnet.microsoft.com/en-us/download/dotnet/6.0 that matched the version already on my Win 2022 server and used that to do the uninstall, then referenced it in my dsc config file. Ran perfectly.

PS, the AZURE Win 2022 servers we have are rolling 6.0.26

I also found that running the DSC, let it fail then repairing .NET and running again worked, BUT, I don't like the act of crashing out, who else know what else could be half cooked.
